jablib/src/main/java/org/jabref/logic/citationstyle/CitationStyleOutputFormat.java

```diff
@@ -5,7 +5,8 @@
 public enum CitationStyleOutputFormat {
 
     HTML("html", OS.NEWLINE + "<br>" + OS.NEWLINE),
-    TEXT("text", "");
+    TEXT("text", ""),
+    MARKDOWN("markdown", OS.NEWLINE + "<br>" + OS.NEWLINE);
```

Comment:
`<br>` is not a good line separator in markdown. Just remove it. Two empty lines are ok.